kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/Filters/__tests__/AddEditFilterContainer.spec.tsx

```diff
@@ -71,26 +71,25 @@ describe('AddEditFilterContainer component', () => {
       userEvent.clear(textAreaElement);
 
       await waitFor(() => {
-        const requiredFieldTextElements =
-          screen.getAllByText(/required field/i);
-        expect(requiredFieldTextElements).toHaveLength(2);
+        const requiredFieldTextElements = screen.getByText(/required field/i);
+        expect(requiredFieldTextElements).toBeInTheDocument();
       });
     });
   });
 
   describe('Custom setup for the component', () => {
     it('should render the input with default data if they are passed', () => {
       setupComponent({
-        inputDisplayNameDefaultValue: mockData.name,
-        inputCodeDefaultValue: mockData.code,
+        inputDisplayNameDefaultValue: '',
```

Comment:
shouldn't we test it here with a mock string as well since this component is one responsible to rendering this output ?

Reply:
@Mgrdich do you have any idea how can I check inputCodeDefaultValue?
I'm trying this way which worked in other test files but here it's not working
      const textAreaElement = inputs[0] as HTMLTextAreaElement;
      expect(textAreaElement.value).toEqual(mockData.code);

Reply:
maybe initialisation is going wrong did you check the edit page ? to see if it renders with values from the start ?

Reply:
that is why it is showing the build error

Reply:
@rAzizbekyan  or maybe there is another way it was not showing in the value , since it uses a a library and that one renders the value on its own pace maybe you can try 
`waitfor` the input to become that value.

Reply:
@Mgrdich I tested, in edit page it receives given default value, but somehow I can't get it inside tests, waitfor didn't work as well